timesketch/lib/datastores/elastic.py

```diff
@@ -415,6 +412,65 @@ def search_stream(self, sketch_id=None, query_string=None,
             for event in result['hits']['hits']:
                 yield event
 
+    # pylint: disable=line-too-long
+    def get_sketch_labels(self, sketch_id, indices):
+        """Aggregate labels for a sketch.
+
+        Args:
+            sketch_id: The Sketch ID
+            indices: List of indices to aggregate on
+
+        Returns:
+            List with label names.
+        """
+        # This is a workaround to return all labels by setting the max buckets
+        # to something big. If a sketch has more than this amount of labels
+        # the list will be incomplete but it should be uncommon to have >10k
+        # labels in a sketch.
+        max_labels = 10000
+        label_aggregation = {
+            "aggs": {
```

Comment:
can you use single quotes, so that we are consistent in the codebase?

Reply:
Good catch, fixed.